scala3/contribute-to-docs.md

```diff
@@ -16,7 +16,7 @@ We welcome contributions from the community to every aspect of the documentation
 
 
 ### How can I contribute?
-In general, there is many different ways you could help us:
+In general, there are many ways you could help us:
```

Comment:
If you don’t mind, please change “could” to “can.” (I prefer a more active tense.)

Reply:
Fixed